src/messages/textDocument_document.cc

```diff
@@ -6,6 +6,7 @@
 #include "query.hh"
 
 #include <algorithm>
+#include <stack>
```

Comment:
I removed this as well. We can just use `vector`.

Reply:
good idea